src/common/refiners/ForwardDateRefiner.ts

```diff
@@ -17,6 +17,27 @@ export default class ForwardDateRefiner implements Refiner {
         results.forEach(function (result) {
             let refMoment = dayjs(context.refDate);
 
+            if (result.start.isOnlyTime() && refMoment.isAfter(result.start.dayjs())) {
+                refMoment = refMoment.add(1, "day");
+
+                result.start.imply("day", refMoment.date());
+                result.start.imply("month", refMoment.month() + 1);
+                result.start.imply("year", refMoment.year());
+
+                if (result.end && result.end.isOnlyTime()) {
+                    result.end.imply("day", refMoment.date());
+                    result.end.imply("month", refMoment.month() + 1);
+                    result.end.imply("year", refMoment.year());
+
+                    if (refMoment.isAfter(result.end.dayjs())) {
+                        refMoment = refMoment.add(1, "day");
+                        result.end.imply("day", refMoment.date());
+                        result.end.imply("month", refMoment.month() + 1);
+                        result.end.imply("year", refMoment.year());
+                    }
```

Comment:
This seems redundant, but it prevents a scenario where the end date lands on the same day that we've adjusted the start date to -- Specifically when the end _**time**_ is earlier in the day then the **_start time_**.

Reply:
Should we compare `refMoment.isAfter(result.end.dayjs())`? or `result.start.dayjs().isAfter(result.end.dayjs())`?

I think the latter is more robust??